modules/mono/editor/GodotTools/GodotTools/Export/ExportPlugin.cs

```diff
@@ -17,6 +17,8 @@ namespace GodotTools.Export
 {
     public partial class ExportPlugin : EditorExportPlugin
     {
+        private List<string> tempFolders = new List<string>();
```

Comment:
Nit: our naming convention is prefix private class fields with an underscore (`_tempFolders`).